vcluster_versioned_docs/version-0.22.0/configure/vcluster-yaml/experimental/native-sleep-mode.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Experimental Sleep Mode
+sidebar_label: sleepMode
+sidebar_position: 7
+sidebar_class_name: pro
+description: Configure a vCluster to sleep/wake without an agent
+---
+
+import ProAdmonition from '../../../_partials/admonitions/pro-admonition.mdx'
+import ExperimentalSleepMode from '../../../_partials/config/experimental/sleepMode.mdx'
+
+<ProAdmonition />
+
+:::warning
+Native sleep mode is intended for pre-production use cases only, and comes with some limitations and
+caveats.
+:::
+
+### Configuration examples
+
+
+
+```yaml title="Sleep after 3 hours of inactivity, anything that does not have the label dont=sleep"
+experimental:
+  sleepMode:
+    enabled: true
+    autoSleep:
+      afterInactivity: 3h  # Uses Go's Duration with a max unit of hour
```

Comment:
Are other values possible here? Like minutes etc? It would be nice to include a callout with link to the Go's Duration fields in :::tip or :::info admonitions.